lib/diagnostic.rb

```diff
@@ -108,10 +108,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person[:favorite_number] = 14'
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = ' person = Hash.new { |hash, key| hash[key] = "#{first_name} is a super-dude." }'
```

Comment:
This would be with the constructor but we were looking for the shorthand syntax, like this:

`'person.default = "#{person[:first_name]} is great"'`